lib/developer_portal/app/controllers/developer_portal/admin/accounts_controller.rb

```diff
@@ -35,7 +35,7 @@ def update
   private
 
   def account_params
-    params.require(:account)
+    params.require(:account).except(*site_account.fields_definitions.by_target('Account').select(&:read_only).map(&:name))
```

Comment:
Same here, could I permit a white list of columns + non read-only params without breaking anything?

Reply:
I don't see the original question. I would suggest though here as well, to integrate the logic into the scope instead of adding the logic everywhere. e.g.
```suggestion  params.require(:account).except(*site_account.fields_definitions.by_target('Account', read_only: false).pluck(&:name))
```

where not specifying `read_only` should result in returning all fields regardless of read_only value